### ISSUE TITLE
XMPP presence priority

### ISSUE THREAD

**Issue by Neustradamus** [credija/opa#38]
Can you add the XMPP presence priority?
from -128 to +127.

**Comment by kevinfaguiar** [comment]
You mean when sending the presence stanzas? I could make it a configurable prop, like the XMPP server URL.

**Comment by Neustradamus** [comment]
From here: https://tools.ietf.org/html/rfc6121#section-4.7.2.3

**Comment by kevinfaguiar** [comment]
> From here: https://tools.ietf.org/html/rfc6121#section-4.7.2.3

Yeah it is sending the priority prop on presence stanzas... will look into a solution for this...

**Comment by kevinfaguiar** [comment]
XMPP presence priority now defaults to +127

**Comment by Neustradamus** [comment]
Why have you selected directly the max value by default?

It is possible to select manually in the webchat?
The user must select a value only between -128 and +127.

**Comment by kevinfaguiar** [comment]
I'm going to need to think about it better, but I do not think of leaving it configurable by the user as it will add more settings and the idea of Opa is not to support everything, but to be a simple web client for XMPP. If I can find a way that looks cool in the user experience I will add this functionality.
So, I'm going to open this issue as I decide whether this request will be met in some way or whether the priority tag will be omitted (where 0 is assumed).
